libs/ui-kit/toggletip/toggletip.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Output, TemplateRef, ViewChild, ViewEncapsulation } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+
+import { MatButtonModule } from '@angular/material/button';
+
+let instanceId = 0;
+
+@Component({
+  selector: 'cz-toggletip',
+  template: `
+    <ng-template #tpl>
+      <div [id]="dialogId" class="cz-toggletip-content" role="dialog" tabindex="-1">
```

Comment:
the [id] can come after class/role etc :)

Reply:
Id should alwasy be (almost) first, even if it is dynamic :) https://www.notion.so/cognizone/Angular-c8f02176425644c093b1f7c33bcccae9?pvs=4#b48d5c9b77584d769b9947da69a97703